read_huge_file/read_huge_file.py

```diff
@@ -0,0 +1,10 @@
+from typing import Generator
+
+
+def read_large_file(file_path: str) -> Generator[str, None, None]:
+    with open(file_path, "r") as file:
+        while True:
+            data = file.read(1024 * 1024)
+            if not data:
+                break
```

Comment:
The provided approach that reads the file in chunks (1MB) and yields each chunk doesn't exactly fit the requirements of reading the file line by line, as specified in the problem statement. This method is useful for certain apps, but it doesn't ensure that each yielded chunk ends at a line boundary, which might split lines across chunks.

Please, rewrite the method using line-by-line reading approach (just iterate over file using for loop)